source/screens/App/components/SideMenuBar.js

```diff
@@ -5,15 +5,11 @@ import Auth from 'shared/api/auth'
 import { MenuItems } from '../shared/const'
 import './SideMenuBar.css'
 
+//TODO: add isActive class to the currently selected link
 const SideMenuBar = ({ menuItems, Router }) => {
-  let newMenuItems = menuItems
```

Comment:
This was from my branch for authentication. Are you intending for users never to logout?

Reply:
We do the log out on the top nav bar now, so we don't need it here in the side bar